scripts/qa_crowbarsetup.sh

```diff
@@ -2262,7 +2262,7 @@ function enable_ssl_generic()
             if [[ ! $(iscloudver 7plus) ]] ; then
                 return
             fi
-        ;;&
```

Comment:
@gosipyan, @aplanas: I didn't realize that we are running `GNU bash, version 3.2.51(1)` on the crowbar nodes. I am using a slightly more recent version (`GNU bash, version 4.3.42(1)`) :smile:. @gosipyan, please add the additional line I mentioned in my earlier comment to your PR.